src/index.html

```diff
@@ -7,6 +7,44 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+
+
+    <div class="calendar">
+      <div class="calendar__day calendar__day--1 calendar__day--tues"></div>
```

Comment:
In my opinion it will be better to add modificator to calendar block, not to the first element.